services/src/main/java/org/keycloak/email/freemarker/FreeMarkerEmailTemplateProvider.java

```diff
@@ -146,11 +146,16 @@ public void sendConfirmIdentityBrokerLink(String link, long expirationInMinutes)
         BrokeredIdentityContext brokerContext = (BrokeredIdentityContext) this.attributes.get(IDENTITY_PROVIDER_BROKER_CONTEXT);
         String idpAlias = brokerContext.getIdpConfig().getAlias();
         idpAlias = ObjectUtil.capitalize(idpAlias);
+        String displayName = idpAlias;
+        if (!ObjectUtil.isBlank(brokerContext.getIdpConfig().getDisplayName())) {
+            displayName = brokerContext.getIdpConfig().getDisplayName();
+        }
 
         attributes.put("identityProviderContext", brokerContext);
         attributes.put("identityProviderAlias", idpAlias);
+        attributes.put("identityProviderDisplayName", displayName);
 
-        List<Object> subjectAttrs = Arrays.asList(idpAlias);
```

Comment:
here you change an attribute which will result in a different output in the event

Reply:
Please notice remarks in this PR [comment 1](https://github.com/keycloak/keycloak/pull/7669/files#r544241890) and [comment 2](https://github.com/keycloak/keycloak/pull/7669/files#r539639446). That is why I didn't remove or change any existing attribute.

Reply:
ah okay than i will remain silent :)!